components/director/pkg/resource/resource.go

```diff
@@ -129,10 +135,12 @@ var tenantAccessTable = map[Type]string{
 	EventSpecFetchRequest:    "event_specifications_fetch_requests_tenants",
 	APISpecification:         "api_specifications_tenants",
 	EventSpecification:       "event_specifications_tenants",
+	CapabilitySpecification:  "capability_specifications_tenants",
 	Document:                 "documents_tenants",
 	BundleInstanceAuth:       "bundle_instance_auths_tenants",
 	API:                      "api_definitions_tenants",
 	EventDefinition:          "event_api_definitions_tenants",
+	Capability:               "capabilities_tenants",
```

Comment:
This view is missing. Maybe it was actually introduced but with other name - `tenants_capabilities`